packages/material-ui/src/NativeSelect/NativeSelect.js

```diff
@@ -156,6 +162,7 @@ NativeSelect.propTypes = {
   children: PropTypes.node,
   /**
    * Override or extend the styles applied to the component.
+   * @default {}
```

Comment:
Probably better to remove this and make the classes object not be required by NativeSelectInput but I wasn't sure.